crates/core/src/error.rs

```diff
@@ -60,13 +60,15 @@ pub enum IndexError {
     IndexAlreadyExists(IndexDef, String),
     #[error("Column not found: {0:?}")]
     ColumnNotFound(IndexDef),
-    #[error("Unique constraint violation '{}' in table '{}': column: '{}' value: {}", constraint_name, table_name, col_name, value.to_satn())]
+    #[error("Unique constraint violation '{}' in table '{}': column(s): '{:?}' value: {}", constraint_name, table_name, cols, value.to_satn())]
```

Comment:
`cols` should probably be called `col_names` considering elsewhere in the code `cols` refers to a list of `id`s